### DISCUSSION TITLE
Cannot copy-paste from GitHub Issue Template (form) placeholder

### DISCUSSION THREAD

**Discussion by pllim** [community/community#45582] [Projects and Issues] [answered]
### Select Topic Area

Product Feedback

### Body

I deployed an issue template using https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema . I put some commands user can run locally on the `placeholder` of a `textarea` to help them fill the form. Unfortunately, user is unable to copy-paste the commands from `placeholder`. And the whole thing disappears when you start typing in it.

**Comment by jge162** [comment, marked as answer]
Hello @pllim The issue with the placeholder disappearing when the user starts typing is a known issue with the <textarea> element in HTML. 

One solution to this problem is to use the ‘value’ attribute to pre-populate the <textarea> with the desired text, rather than using the placeholder attribute. This way, the text will remain visible even when the user starts typing.

**Comment by pllim** [comment]
Then what is `placeholder` really for? Why not everyone just use `value`? Is copy-paste a planned feature or a "won't fix"? Thanks for the quick response!

**Reply by jge162** [reply]
The ‘placeholder’ attribute is intended to provide a brief hint or explanation of the expected format or content of an input field, such as a text area or text input.

The ‘value’ attribute, on the other hand, is used to set the default value of the input field, which will remain visible even when the user starts typing.

**Comment by pllim** [comment]
OK, thanks!
